### ISSUE TITLE
General discussion

### ISSUE THREAD

**Issue by joe-no-body** [joe-no-body/beyond-spreadsheets#2]
I propose we move the running discussion from the [Gist comments](https://gist.github.com/joe-no-body/003a12e1f70b908bd57b3887de44942d) to this issue.

**Comment by joe-no-body** [comment]
A couple logistical proposals:

1. Let's keep any POC work in separate repos for now.
2. The wiki is good for discussion and brainstorming, but as ideas start to solidify I propose that we commit them and share via PR. We've been doing a lot of line-by-line reviews on each other's ideas that I think would be clearer in that format.

**Comment by rneilson** [comment]
Agreed on both counts, esp 2 (I keep tripping up on GH wikis not having PRs)

**Comment by joe-no-body** [comment]
Cool. As an addendum, I think anything that's going to be PR'd should have an issue to track it or at least have the PR opened as early as possible for visibility. Likewise, when we start breaking out tasks, let's explicitly assign them so we know who's working on it.

That said, I'm going to try and force myself to actually POC something now.

**Comment by joe-no-body** [comment]
For visibility I'll drop the link to my POC here: https://github.com/joe-no-body/transducer/pull/1 I'm using the transducer name 'cause right now it's just a gross JSON-based PL and I don't really want it to be actual code we use later.
